include/openssl/ec.h

```diff
@@ -729,6 +729,7 @@ int ECPKParameters_print_fp(FILE *fp, const EC_GROUP *x, int off);
 /* some values for the flags field */
 # define EC_FLAG_NON_FIPS_ALLOW  0x1
 # define EC_FLAG_FIPS_CHECKED    0x2
+# define EC_FLAG_EXT_PKEY        0x4
```

Comment:
Why this name for this flag? It doesn't seem to intuitively describe what it does

Reply:
For RSA, it's a bit of a mess. There are two flags - RSA_FLAG_EXT_PKEY on struct RSA and RSA_METHOD_FLAG_NO_CHECK is on RSA_METHOD needed to make a dummy RSA private key actually work. For EC, I believe we can get away with just one flag on the EC_KEY struct itself, and I chose EC_KEY_EXT_PKEY for parity.